activiti-core-common/activiti-expression-language/src/main/java/org/activiti/core/el/ELContextBuilder.java

```diff
@@ -52,6 +52,7 @@ public ELContextBuilder withVariables(Map<String, Object> variables) {
 
     public ELContext build() {
         CompositeELResolver elResolver = createCompositeResolver();
+        elResolver.add(new EnvironmentVariableELResolver());
```

Comment:
I order to make it easier to modify in the future and to make it possible to add resolvers from outside (Open/Closed principle),  I think it's better add an extension point to provide extra resolvers. You could then, move `EnvironmentVariableELResolver` inside `activiti-spring-boot-starter` and instantiate it as a bean conditional on a property (`spring.activiti.env-var-el-resolver.enabled`) that is disabled by default to avoid potential conflicts on processes that have a process variable named `vars`. 

Have a look on how it was done for `CustomFunctionProvider`: you can have something similar for resolvers (`public ELContextBuilder withCustomResolvers (List<ELResolver> customResolvers)`).